android/tangram/jni/platform_android.cpp

```diff
@@ -266,5 +276,16 @@ void setCurrentThreadPriority(int priority) {
     setpriority(PRIO_PROCESS, tid, priority);
 }
 
+void featureSelectionCallback(JNIEnv* jniEnv, const std::vector<std::shared_ptr<Tangram::Properties>>& items) {
+
+    jlong jresult = 0;
+    jclass propertiesClass = jniEnv->FindClass("com/mapzen/tangram/Properties");
+
+    jresult = reinterpret_cast<jlong>(new std::shared_ptr<Tangram::Properties >(items[0]));
```

Comment:
So, it will always return the properties for the "first" feature being picked?

Should we be sending a list all features picked by our code?